src/components/CoinbasePayBtn.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useRef, useState } from "react"
+import CnButtonImg from "../assets/button-cbPay-compact-addCrypto.png"
+import { initOnRamp } from "@coinbase/cbpay-js"
+import { styled } from "@mui/material"
+
+const CnPayButton = styled("button")(() => ({
```

Comment:
rename to CoinbasePayButton or StyledCoinbaseBtn or something similar. What is Cn?

Reply:
was going to type type `CbPayButton` referring the `coinbase pay button`